alfresco-transform-tika/alfresco-transform-tika/src/main/resources/IPTCMetadataExtractor_metadata_extract.properties

```diff
@@ -0,0 +1,308 @@
+#
+# IPTCMetadataExtracter - default mapping
+#
+# author: David Edwards
+
+# TODO Complete mappings (currently copied from other files)

```

Comment:
Can I assume this TODO is captured in one of the related JIRA tasks (as per the two additional one below).

Reply:
Yes it's related to ATS-900